egui_glow/Cargo.toml

```diff
@@ -23,19 +23,26 @@ all-features = true
 
 [dependencies]
 egui = { version = "0.15.0", path = "../egui", default-features = false, features = ["single_threaded"] }
-egui-winit = { version = "0.15.0", path = "../egui-winit", default-features = false, features = ["epi"] }
-epi = { version = "0.15.0", path = "../epi", optional = true }
 
+epi = { version = "0.15.0", path = "../epi", optional = true }
 glow = "0.11"
-glutin = "0.27"
 memoffset = "0.6"
 
+[target.'cfg(not(target_arch = "wasm32"))'.dependencies]
+egui-winit = { version = "0.15.0", path = "../egui-winit", default-features = false, features = ["epi"] }
+glutin = "0.27"
+
+
+[target.'cfg(target_arch = "wasm32")'.dependencies]
+web-sys = { version = "0.3", features=["HtmlCanvasElement", "WebGl2RenderingContext","WebGlRenderingContext", "Window","console"] }
+wasm-bindgen = { version = "0.2" }
+
 [dev-dependencies]
 image = { version = "0.23", default-features = false, features = ["png"] }
 
 [features]
 default = ["clipboard", "default_fonts", "links", "persistence"]
-
+painter_only = []
```

Comment:
`painter_only` is now a feature that _removes_ features, which is… weird :) 

Better to call it something like `winit` to opt-in to the winit-parts of `egui_glow`

Also: the features are sorted by name to make them easier to find. And there are spaces between them to help readability.

Reply:
add winit feature.